src/core/layout/gel-button.js

```diff
@@ -19,10 +19,11 @@ export class GelButton extends Phaser.GameObjects.Container {
     constructor(scene, x, y, config) {
         const metrics = getMetrics();
         super(scene, x, y);
-
         this.sprite = scene.add.sprite(0, 0, assetPath(Object.assign({}, config, { isMobile: metrics.isMobile })));
         this.setScrollFactor(0);
-        this.sprite.setScrollFactor(0);
+        if (!config.inScrollable) {
```

Comment:
Might be nice to make this more generic sounding rather than tying its name to the scrollable list - I think we might need it elsewhere so maybe just "scrollable" ?

Also feel free to use shortcutting to keep the line count down like:

        `!config.scrollable && this.sprite.setScrollFactor(0);`